mediaire_toolbox/transaction_db/transaction_db.py

```diff
@@ -19,6 +54,18 @@ def __init__(self, engine):
         DBSession = sessionmaker(bind=engine)
         self.session = DBSession()
         create_all(engine)
+        db_version = self.session.query(SchemaVersion).get(SCHEMA_NAME)
+        if not db_version:
+            # first time we see the schemaversion table
+            # we don't know what's the current version
+            db_version = SchemaVersion()
+            db_version.schema_version = 1
+            self.session.add(db_version)
+            self.session.commit()
+            migrate(self.session, db_version, errors_allowed=True)
```

Comment:
We allow errors when the version table doesn't exist cause we don't know at which version the database is, so some migrations might fail because of applying duplicate ALTER TABLE statements, for instance